requirements_all.txt

```diff
@@ -358,6 +359,9 @@ pyenvisalink==1.7
 # homeassistant.components.ifttt
 pyfttt==0.3
 
+# homeassistant.components.light.mipow
+pygatt[GATTTOOL]==3.0.0
```

Comment:
This will not work on Fedora and I guess also not on CentOS as it requires the devel packages for Bluetooth. We are in the same boat as for the bluettooth tracker.